lib/actions/apiV2.js

```diff
@@ -1011,6 +1011,24 @@ export function routingQuery(searchId = null, updateSearchInReducer) {
                   response.data?.plan?.routingErrors.filter(
                     (re) => re?.code !== 'NO_TRANSIT_CONNECTION'
                   )
+
+                // Add accessibility error, if it is turned on
+                const state = getState()
+                const { displayA11yError } = state.otp.config?.itinerary
+                if (displayA11yError) {
+                  if (
```

Comment:
this is a lot of if statements maybe use `&&`?

Reply:
I think it's cleaner like this!